docs/node_benchmarks.md

```diff
@@ -0,0 +1,351 @@
+# Node Benchmarks for Scylla
+
+Whether you're running performance jobs or investigating an issue, you might
+want to now if the hardware Scylla is running meets you baseline perfromance
+expectations. That's what node benchmarks are for.
```

Comment:
`if the hardware Scylla is running meets you baseline perfromance expectations.` ->
if the hardware Scylla is running **ON** meets you**R** baseline per**FOR**mance expectations.

Reply:
Fixed